manifests/resolved.pp

```diff
@@ -181,7 +181,7 @@
     false   => 'no',
   }
 
-  if $cache {
+  if $_cache {
```

Comment:
Does this change actually make sense? `$cache` is a boolean, `$_cache` is a string.

Reply:
Well. If you set `$cache = false`. The ini_setting is not executed and `cache = no` is not set.